spec/requests/posts_request_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    before(:example) { get '/users/1/posts' }
+    it 'responds with the code 200' do
+      expect(response).to have_http_status(:ok)
+    end
+    it "renders 'index' template" do
+      expect(response).to render_template(:index)
+    end
```

Comment:
- [ ] As stated by the previous reviewer, your tests that render templates are failing because you are missing this gem, 'rails-controller-testing'. Kindly include the gem in your Gemfile, run `bundle install` and run the requests tests again.

  ```
  gem 'rails-controller-testing'
  ```

  ![rend](https://user-images.githubusercontent.com/81468675/186419939-cec900ac-9ea2-4466-aaac-237f9a340968.jpg)

Reply:
hello @julie-ify, I did it but if something goes wrong with it you can contact me on slack, please!

Reply:
Well resolved 👍🏽